lib/beaker/cli.rb

```diff
@@ -113,6 +113,15 @@ def execute!
         end
       #cleanup phase
       rescue => e
+        begin
+          if @options[:fail_mode].to_s =~ /slow/
```

Comment:
Wasn't sure if the pre-cleanup should ignore the slow fail mode or not. Maybe it should, since I'm billing that it will "always run"?